### ISSUE TITLE
Add back in user @ mentions

### ISSUE THREAD

**Issue by sblaplace** [TieSoul/TieBot#4]
Have bot @ mention users when replying to bot mentions

**Comment by facets** [comment]
Yes, so I can clone the bot, log in the evil twin, and then have the evil one @ mention the other? I can see this going perfectly... :3

**Comment by sblaplace** [comment]
@facets You have a fair point :P Maybe, after we implement replying to @ mentions, we check for mentioning loops? Not sure how we'd go about that
